Guide/tailwindcss.markdown

```diff
@@ -92,18 +92,47 @@ We also need a CSS entry point for Tailwind. Create a new file at `tailwind/app.
 }
 ```
 
-### Adding the build step
+### Integrating with `devenv`
 
-We need to add a new build command for starting a tailwind build process to our `Makefile`. For that append this to the `Makefile` in your project:
+To start the TailwindCSS build process with the IHP dev server, add the following to your flake.nix:
 
-```makefile
-tailwind-dev:
-	tailwindcss -c tailwind/tailwind.config.js -i ./tailwind/app.css -o static/app.css --watch
+```nix
+...
+ihp = {
+    enable = true;
+    projectPath = ./.;
+    packages = with pkgs; [
+        # Native dependencies, e.g. imagemagick
+        (nodePackages.tailwindcss.overrideAttrs
```

Comment:
Maybe worth adding a comment?

```suggestion
        # Use nix to download tailwind and related packages from npm.
        (nodePackages.tailwindcss.overrideAttrs
```

Reply:
This part was mentioned in the prior section "Adding package to flake.nix", so would the comment be redundant here?

I struggled a bit on the decision to include that snippet here, but in the end I feel including that snippet here makes it easier for users to find where to add the new configuration relative to all the other stuff in `flake.nix`.